common/src/main/java/org/jboss/hal/testsuite/util/ConfigChecker.java

```diff
@@ -154,10 +167,20 @@ public Builder configFragment(ConfigFragment config) {
          * multiple times. The actual edit will be performed as soon as client calls the {@link #andSave()} </b>
          */
         public Builder edit(InputType inputType, String identifier, Object attrValue) {
+            return edit(inputType, identifier, attrValue, InputMethod.MACHINE);
+        }
+
+        /**
+         * use this method only if you need to set explicit {@link InputMethod}. Set field identified by
+         * <b>{@code identifier}</b> to be edited with <b>{@code attrValue}</b>. Can be called multiple times. The
+         * actual edit will be performed as soon as client calls the {@link #andSave()} </b>
+         * @param inputMethod Input method to be used. For example InputMethod.HUMAN to edit form like human.
```

Comment:
I would suggest to extend comment to emphasise that using this method is meant just as a workaround to observed weird behaviour of some input fields and should be used only in cases when standard `edit(InputType inputType, String identifier, Object attrValue)` method is not working properly.